core/sql/common/Int64.cpp

```diff
@@ -136,6 +140,22 @@ void convertInt64ToAscii(const Int64 &src, char* tgt)
   strcpy(tgt, s);
 }
 
+void convertUInt64ToAscii(const UInt64 &src, char* tgt)
+{
+  UInt64 temp = src;
+  char buffer[21];
+  char *s = &buffer[21];
+  *--s = '\0';
+  do {
+    char c = (char) (temp % 10);
+    if (c < 0)
+      c = -c;
```

Comment:
I'm trying to imagine why c would ever be negative. Seems like since temp is unsigned, we'd always see 0, ... 9 as possible values from temp % 10?